remove reference to unmaintained eEVM

Otto-AA opened the pull request:
<!--- Provide a general summary of your changes in the Title above -->

## Description

[eEVM](https://github.com/microsoft/eEVM) is archived and hasn't been updated in a year. I did not find any link to a new code location, so I think this project is not maintained in any more.

This PR removes the link from this page: https://ethereum.org/en/developers/docs/evm/

<!--- Describe your changes in detail -->

## Related Issue

<!--- This project accepts pull requests related to open issues -->
<!--- If suggesting a new feature or change, please discuss it in an issue first -->
<!--- If fixing a bug, there should be an issue describing it with steps to reproduce -->
<!--- Please link to the issue here: -->

nhsz reviewed (changes requested):
Thanks @Otto-AA. We would also need to remove the link from the translations (check the `/public/content/translations/` path or just look or the url)

Otto-AA commented:
Thanks for the remark, forgot about this.

I've removed eEVM from the translations now and also added revm to a few translations where it was missing.

nhsz reviewed (approved):
LGTM, thanks!
